ddp-workspace/projects/ddp-sdk/src/lib/components/confirmDialog/confirmDialog.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="confirm-dialog">
+    <span class="confirm-dialog-title"
+          [innerHTML]="title | translate">
+    </span>
+    <div class="confirm-dialog-buttons">
+        <button mat-flat-button class="ctrl-btn confirm-btn"
+                color="warn"
```

Comment:
are we sure that confirm button should be in red color?

Reply:
I am not. 
However, the dialogs for now notify about **_Remove_**/**_Re-upload_** a file. 
They are needed to highlight that the operation can not be undo.

Reply:
Do you have some suggestions about it ?

Reply:
well, the color might be configurable I guess, but it might be extended later if we feel that the red color is the right one here

Reply:
@aglushko-broad, which color are you suggesting to have there by default ?

Reply:
Another thing is that the confirmation dialog is used in other places (i.g. modalActivityBlock component)

Reply:
I don't have any suggestion, my comment initially was rather about "should we just copy it from original dialog or maybe even discuss with the designer what's better here" but as long as I'm not a designer I can't say which color might be better here. As I already wrote in the previous comment if red sounds right for you feel free to do nothing and ignore the comment. It wouldn't be a problem to add config functionality in the future if we need it.

Reply:
Ok, then I propose leave it as is for now because we do not have any comprehension how it will be used later.

Reply:
Made the button color configurable and set the default 'primary' theme color palette.
![image](https://user-images.githubusercontent.com/7396837/120460231-9a407a80-c3a1-11eb-8e11-812688b1e4dc.png)

Eventually, we can override the color in specific study css if needed.